diff --git a/lib/agents/hermes.js b/lib/agents/hermes.js
index 7190f39be6354b4f0f603aca78192a8512eafc13..23c19b3c3e12b8eb6036967b96b33dd2089c88cc 100644
--- a/lib/agents/hermes.js
+++ b/lib/agents/hermes.js
@@ -35,7 +35,7 @@ class HermesAgent extends ConsoleAgent {
       args.unshift('-non-strict');
     }
 
-    args.unshift('-Xintl', '-enable-eval', '-fenable-tdz');
+    args.unshift('-Xintl', '-enable-eval');
 
     // There is currently no flag for supporting modules in Hermes
     // if (options.module && this.args[0] !== '-m') {
